src/main/resources/testdata/07_applications.sql

```diff
@@ -0,0 +1,45 @@
+-- =============================================
+-- 07_applications.sql
+-- Inserts example applications
+-- Preconditions:
+--   - applicants must exists
+-- =============================================
+
+
+-- clean existing data
+DELETE
+FROM applications
+WHERE application_id IN ('1c982d01-4560-437a-92d8-310aacb5991c',
+                  'd6c49666-2392-4184-ba12-2a0074b2d138',
+                  '2438c612-6045-408c-8528-af4e99a52859',
+                  '2a18ee8b-9f8d-4369-89fd-7d4768845efa',
+                  '6365bb47-38f9-4d79-9c3a-4f1c094d413c',
+                  '4c05bf7d-f06c-44da-982b-7760fa49771c',
+                  'db7fdef7-37f6-474b-a646-9068da9135ca',
+                  'f7ec007f-fba0-4135-b88c-c70eec79faf3',
+                  'bce85235-25b7-46c2-988c-9290be79dc57',
+                  'cafc966e-1927-4593-bd36-986e8f06db25');
+
+-- insert test data
+REPLACE INTO applications (application_id, applicant_id, job_id, application_state, desired_start_date, motivation, rating, created_at, last_modified_at)
```

Comment:
Manually adding a created_at and last_modified_at column to the test data results in an error in the applications table.
```suggestion
REPLACE INTO applications (application_id, applicant_id, job_id, application_state, desired_start_date, motivation, rating)
```